addons/overpressure/initSettings.inc.sqf

```diff
@@ -1,7 +1,15 @@
 private _category = [LELSTRING(common,categoryUncategorized), LLSTRING(DisplayName)];
 
 [
-    QGVAR(distanceCoefficient), "SLIDER",
+    QGVAR(backblastDistanceCoefficient), "SLIDER",
+    [ELSTRING(statBackblastRange,Overpressure), LSTRING(distanceCoefficient_toolTip)],
+    _category,
+    [0, 10, 1, 1],
+    1
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(overpressureDistanceCoefficient), "SLIDER",
     [LSTRING(distanceCoefficient_displayName), LSTRING(distanceCoefficient_toolTip)],
     _category,
     [-1, 10, 1, 1],
```

Comment:
If both settings work in the same way, then the lower bound should be adjusted to 0 as well.